### PR TITLE
New apps related webhooks events added to enums.

### PR DESCRIPTION
**Pull request by SzymJ**
PR with a change [here.](https://github.com/saleor/saleor/pull/9698)

### CODE REVIEW

**Review by fowczarek — APPROVED** [review]

File: docs/developer/extending/apps/asynchronous-webhooks.mdx [hunk 1]
```diff
@@ -87,6 +87,10 @@ Webhooks can be used to receive data from Saleor when particular events happen.
 | Name                           | Description                                                                                                                                                                                                             |
 | -------------------------------| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `ANY_EVENTS`                   | Receive webhooks when any of the available events is triggered (wildcard event).                                                                                                                                        |
+| `APP_CREATED`                  | A new app created.                                                                                                                                                                                                      |
```
Comment (patrys) [inline docs/developer/extending/apps/asynchronous-webhooks.mdx:90]: Are apps created or installed?

Comment (SzymJ) [reply docs/developer/extending/apps/asynchronous-webhooks.mdx:90]: we send this event when an app is created and after `successful installation/reinstallation`

Comment (patrys) [reply docs/developer/extending/apps/asynchronous-webhooks.mdx]: I'd rather emit `APP_INSTALLED` for both cases.

Comment (SzymJ) [reply docs/developer/extending/apps/asynchronous-webhooks.mdx]: done, PR with change [here.](https://github.com/saleor/saleor/pull/9757)

**Review by maarcingebala — APPROVED** [review]

**Pull request merged**